app/assets/javascripts/application.js.erb

```diff
@@ -388,6 +388,8 @@ var _Lobsters = Class.extend({
 var Lobsters = new _Lobsters();
 
 $(document).ready(function() {
+  Lobsters.curUser = parseInt($('body').data('current-user-id'));
```

Comment:
Since ES5 the radix defaults to `10`, even if the input string starts with `0`. [MDN still recommends it with string wording](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt), but that hasn't been updated since 2013. As far as I know, practically all browsers in current use (including IE11) behave like ES5 says they should, and there is no longer a reason to explicitly specify it.

That being said, it doesn't hurt to add it I suppose, but I mostly stopped doing so recently myself.